### PR TITLE
Docs: organization docs refactor

### PR DESCRIPTION
**Pull request by chri2547**
This PR reorganizes the Grafana organization docs.

### CODE REVIEW

File: docs/sources/administration/manage-organizations/_index.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+  - dashboards
+---
+
+# Managing organizations
+
+This topic describes what organizations are, and how to create, edit, and delete them.
+
+## About organizations
+
+An organization is an entity that helps you limit the dashboards that users can see. Their purpose is to provide completely separate experiences, which look like multiple instances of Grafana, within a single instance. Multiple organizations are easier and cheaper to manage than multiple instances of Grafana.
```
Comment (mjseaman) [inline docs/sources/administration/manage-organizations/_index.md:18]:
```suggestion
An organization is an entity that helps you isolate users and resources like dashboards, annotations, and data sources from each other. Their purpose is to provide completely separate experiences, which look like multiple instances of Grafana, within a single instance. Multiple organizations are easier and cheaper to manage than multiple instances of Grafana.
```

File: docs/sources/administration/manage-organizations/_index.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+To an organization you can add teams and dashboards. The member of one organization cannot view dashboards assigned to another organization. However, a user can belong to multiple organizations.
```
Comment (mjseaman) [inline docs/sources/administration/manage-organizations/_index.md:20]:
```suggestion
Users, configuration settings, and Grafana Enterprise licenses are shared between organizations. Other resources, like dashboards, data sources, annotations, folders, Teams, and Alerts, are isolated within each organization and cannot be easily shared with another organization. The member of one organization cannot view dashboards assigned to another organization. However, a user can belong to multiple organizations.
```

File: docs/sources/administration/manage-organizations/_index.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+1. Sign in to Grafana as a server administrator.
+1. Hover your cursor over the **Server Admin** (shield) icon until a menu appears, and click **Orgs**.
+
+A list of organizations appears.
+
+![Server Admin organization list](/static/img/docs/manage-users/server-org-list-7-3.png)
+
+## Create an organization
+
+Create an organization when you want to segment dashboards among Grafana users.
```
Comment (mjseaman) [inline docs/sources/administration/manage-organizations/_index.md:43]:
```suggestion
Create an organization when you want to isolate dashboards and other resources from each other.
```

**Comment by mjseaman** [comment]
Looks good! You might consider adding just the Orgs column from [this table](https://docs.google.com/document/d/1UdpQ8IofYIfEM0un5j-q0LbX3xBRS9kUkTWO8hDSxpc/edit#bookmark=id.w87v8as8nea) as a way to clearly indicate what resources are shared and isolated between one org and another.

**Review by mjseaman — APPROVED** [review]
LGTM!

**Comment by torkelo** [comment]
@chri2547 I don't see aliases for the two removed files. this will break old links / SEO

**Pull request merged**